versions/3.0.3.md

```diff
@@ -387,7 +387,7 @@ The following shows how variables can be used for a server configuration:
       "variables": {
         "username": {
           "default": "demo",
-          "description": "this value is assigned by the service provider, in this example `gigantic-server.com`"
+          "description": "A user-specific subdomain provisioned for each account. Use `demo` for a free sandbox environment."
```

Comment:
Avoiding "assigned by the service provider," which might be misinterpreted in some way similar to my own muddled confusion.